helpers/wrapper/reports/SECURITY.md

```diff
@@ -1,150 +1,80 @@
-- **This invariant currently holds as-is.**
-
-#### 6. Only the sender specified on an order may execute Wrapper.swap.
-
-- Because the sender has authorized the wrapper on the swap contract, the wrapper requires that the sender of the transaction be the same as the order sender.
+- By inspection and testing, line 75 requires that the order signature `v` value is not zero, which is also the value checked by the Swap contract to determine whether to check the signature.
 - **This invariant holds as-is.**
 
-#### 6. The contract is safeguarded against re-entrancy attacks that can make unauthorized or unexpected ether or token transfers.
-
-- Confirmed through manual inspection walking through several attack scenarios. Prior to Swap.swap being called, the transfers use a well-known WETH contract that does not unexpectedly hold balances. It will always REVERT on failures (see WETH audit for notes). After the first conditional in Wrapper.swap, the Swap.swap re-entrancy guards prevent duplicate orders from being submitted and filled. In addition, the fallback function locks ether transfers to only the WETH smart contract.
-- **This invariant currently holds as-is.**
-
-#### 8. Sending the same parameters twice will have the same guarantees as the Swap contract.
-
-- Calls to Wrapper.swap are idempotent. When called many times, the beginning and end state are the same due to having either swapped WETH or transferred ether out of the contract within each transaction.
-- **This invariant currently holds as-is.**
+### Every `swap` through a Wrapper must be transacted by the sender specified on the order.
 
-#### 9. If WETH is not specified on either side of a trade the contract should effectively pass through to Swap.swap.
-
-- In this scenario, none of the conditional statements for Wrapper.swap are hit and thus only swapContract.swap method is called.
-- **This invariant currently holds as-is.**
-
-#### 10. If ether is sent to the contract by anything other than the WETH contract it should REVERT.
-
-- When the contract withdraws from WETH, the WETH contract sends it ether, which is then transferred to the message sender. Otherwise in the fallback function, any message sender other than the WETH address will REVERT.
-- **This invariant currently holds as-is.**
-
-#### 11. Wrapper should not have any impact on affiliates specified to receive WETH.
-
-- In this scenario, an affiliate that is specified to receive WETH will receive WETH if Swap.swap succeeds. There is no wrapping or unwrapping for affiliate parties.
-- **This invariant currently holds as-is.**
-
-## Analysis
-
-#### Slither (static analyzer)
+- By inspection and testing, line 70 requires that the message sender (transaction originator) is the same as the order sender wallet.
```

Comment:
message sender != transaction originator
if using a multisig, the message sender would be the multisig, but the transaction originator would be an EOA